deploy/values.yml

```
@@ -0,0 +1,92 @@
+replicaCount: ${REPLICA_COUNT}
```

I know I have it set as a var somewhere else, but I've now realized it's best to avoid that:
```suggestion
replicaCount: 1
```